packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -155,6 +155,14 @@ export interface TriggerRebase {
 	type: "rebase";
 }
 
+/**
+ * @internal
+ */
+export interface Rollback {
+	type: "rollback";
```

Comment:
The value here gets written to files that are intended to be relatively human-readable. I think we should make the type reflect the operation's real semantics:
```suggestion
	type: "applyThenRollback";
```

Reply:
this will require some other changes to the PR, so probably should make them in an IDE and resolve afterward.